### DISCUSSION TITLE
Not able to install home brew

### DISCUSSION THREAD

**Discussion by JagatheeshMenon** [Homebrew/discussions#1477] [Getting started]
Hi all,

I am finding it difficult to install home brew. 
Please find below error that i am getting when installing the homebrew.

==> Tapping homebrew/core
remote: Enumerating objects: 958747, done.
remote: Counting objects: 100% (531/531), done.
remote: Compressing objects: 100% (230/230), done.
error: 2835 bytes of body are still expected.16 MiB | 3.33 MiB/s
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: index-pack failed
Failed during: git fetch --force origin refs/heads/master:refs/remotes/origin/master

Any suggestion would be of great help.

**Comment by carlocab** [comment]
Most likely a connection issue. See #666 for some suggestions.
